aiidalab_widgets_base/computational_resources.py

```diff
@@ -852,19 +852,26 @@ def _configure_computer(self, computer: orm.Computer):
 
         return True
 
+    def _run_things_on_existing_computer(self, label):
+        """Run things on an existing computer"""
+        try:
+            orm.Computer.objects.get(label=label)
+            for function in self._on_setup_computer_success:
+                function()
+        except common.NotExistent:
+            return False
+        else:
+            return True
+
     def on_setup_computer(self, _=None):
         """Create a new computer."""
         if self.label.value == "":  # check hostname
             self.message = "Please specify the computer name (for AiiDA)"
             return False
-        try:
-            computer = orm.Computer.objects.get(label=self.label.value)
-            self.message = f"A computer called {self.label.value} already exists."
```

Comment:
This line is missing in the new function.

Reply:
I've just put it back to a more appropriate place 👍